Collector/MigrationsCollector.php

```diff
@@ -29,7 +30,13 @@ public function collect(Request $request, Response $response, \Throwable $except
         $planCalculator = $this->dependencyFactory->getMigrationPlanCalculator();
         $statusCalculator = $this->dependencyFactory->getMigrationStatusCalculator();
 
-        $executedMigrations  = $metadataStorage->getExecutedMigrations();
+        try {
+            $executedMigrations  = $metadataStorage->getExecutedMigrations();
+        } catch (Exception $dbalException) {
+            // Ignore connection error and do not collect any data
```

Comment:
Should this at least be logged with
```
$this->dependencyFactory->getLogger()->error(
    'error while trying to collect executed migrations',
    ['exception' => $dbalException]
);
```

Alternatively, have you considered collecting the exception for later display inside the collector?

Reply:
Yes, I was considering logging, but didn't want to add the additional dependency into the class.